qatrack/settings.py

```
@@ -691,7 +690,36 @@ def EXPLORER_PERMISSION_VIEW(request):
 # local_settings contains anything that should be overridden
 # based on site specific requirements (e.g. deployment, development etc)
 
-from .local_settings import *  # noqa: F403, F401, E402
+import os
+if os.environ.get('USE_DOCKER') == 'true':
+    ALLOWED_HOSTS = ['*']
+
+    SECRET_FILEPATH = os.path.join(PROJECT_ROOT, '..', 'deploy', 'docker', 'user-data', 'secret_key.txt')
+    try:
+        with open(SECRET_FILEPATH, 'r') as f:
```

Ruff (0.15.14)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)